collect_app/src/main/java/org/odk/collect/android/utilities/TextUtils.java

```diff
@@ -86,14 +88,29 @@ private static String markdownToHtml(String text) {
         return text;
     }
 
-    public static CharSequence textToHtml(String text) {
+    public CharSequence textToHtml(String text) {
 
         if ( text == null ) {
             return null;
         }
 
-        return Html.fromHtml(markdownToHtml(text));
+        return htmlWrapper.toHtml(markdownToHtml(text));
+
+    }
+
+    public static class HtmlWrapper {
+        public Spanned toHtml(String text) {
+            return Html.fromHtml(text);
+        }
+    }
```

Comment:
Changing the API like this smells unless we intend to make it a convention for similar. I suspect we would use Roboelectric or make it an instrumentation test.